modules/nf-core/bwa/index/main.nf

```diff
@@ -11,20 +11,21 @@ process BWA_INDEX {
     tuple val(meta), path(fasta)
 
     output:
-    tuple val(meta), path(bwa) , emit: index
-    path "versions.yml"        , emit: versions
+    tuple val(meta), path("bwa/") , emit: index
+    path "versions.yml"           , emit: versions
```

Comment:
I'd rather we keep that the way it was for more consistency between the different aligners

Reply:
fine by me! as long as it works